drupal-org.make

```diff
@@ -6,6 +6,7 @@ projects[features] = 3.2
 projects[config_update] = 1.3
 projects[entity] = 1.0-alpha4
 projects[datalayer] = 1.x-dev
+projects[datalayer][patch] = "https://www.drupal.org/files/issues/2857270-4-fix-entity-reference-field-using-name-property.patch"
```

Comment:
Should be followign format:
```
projects[datalayer][subdir] = contrib
projects[datalayer][version] = 1.x-dev
projects[entity_browser][patch][] = "https://www.drupal.org/files/issues/2857270-4-fix-entity-reference-field-using-name-property.patch"
```
because right now with `[patch]` you just override `1.x-dev` value.